src/main/java/Event.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Event class that specifies an event happening at a certain time.
+ */
+public class Event extends Task {
+
+    private String timeslot;
```

Comment:
Should `timeslot` be in camelCase?

Reply:
I agree with this. variable names should be in camelCase

Reply:
Agreed!